### ISSUE TITLE
No Model created

### ISSUE THREAD

**Issue by E104** [Unity-Technologies/ml-agents#1599]
Hi there!

I am trying to train a build of the Balance-Ball-Example, but I have some problems.
I made a build of the scene and saved it into the ml-agents folder.
Than I opened Anaconda and used

`mlagents-learn ../config/trainer_config.yaml --env=balance/BalanceBall --run-id=firstRun --train`

to run the training (the executable is "BalanceBall.exe" and the Project is "balance").
It seems to work - I see a fancy written Unity.Logo and there is a small window with a lot of falling balls.
But after running this for some hours and stopping it with Ctr. + C, I got this strange Output:

![errormessage_large](https://user-images.githubusercontent.com/10851404/51119208-861c7280-1812-11e9-9679-69bf25335fd8.PNG)

I would expect someting like this:
`INFO:mlagents.trainers: firstRun-0: 3DBallLearning: Step: 1000. Mean Reward: 1.242. Std of Reward: 0.746. Training.` Like it's written in the tutorials. But there is just what I posted above. Also the created folder "firstRun-0" in "models" is empty.

Any idea, what I missed?

Ah jeah - and for some reason CPU is used instead of GPU, but I am sure, I installed tensorflow-gpu.

Thank you for your help, guys!

**Comment by vincentpierre** [comment]
This might be the known [bug](https://github.com/Unity-Technologies/ml-agents/issues/980) on windows that Ctrl-C does not save the model. This bug has been addressed in __v0.6.0a__. You will need to update your python code.

**Comment by kiwidave72** [comment]
this isnt a problem with the Ctrl-C bug...it happens on the v0.6.0a release.

theres not output during the training process at all...nothing like 
INFO:mlagents.trainers: firstRun-0: 3DBallLearning: Step: 1000. Mean Reward: 1.242. Std of Reward: 0.746. Training.
this is a major issue as nothing can be trained.

**Comment by E104** [comment]
Yep, I think kiwidave72 is right. Anyway I tried out multible Python-Versions (3.6.0 & 3.6.8) but I hab no luck.

Great thanks for your effort, guys!
If somebody has additional ideas whats going wron on my mashine - I would be very happy!

**Comment by WhiteWolf93** [comment]
I'm having exactly the same problem too yet I cant find any solution, if you find a solution please post it!

**Comment by vincentpierre** [comment]
I am unable to reproduce this bug. My guess is that the environment is not communicating properly after initialization. Could you try in Editor and with the `--slow` argument ? I would like to know if the platforms are indeed being controlled by the external python process. Please make sure that the hyperparameters in the `trainer_config.yaml` file have not been modified. It could be the case that the Learning Brain set to control in the Broadcast Hub of the Academy is not the same as the learning brain attached to each of the agents.

**Comment by wicaodian** [comment]
How dumb that was Oh God
@vincentpierre `It could be the case that the Learning Brain set to control in the Broadcast Hub of the Academy is not the same as the learning brain attached to each of the agents.`

that was the issue now everything seems perfectly fine.

**Comment by wicaodian** [comment]
@E104 
Go to your agent and make sure they are also assigned a learning brain not a player brain and there you go  issue resolved
